uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/api/v3/TaskTest.php

```diff
@@ -1,69 +1,75 @@
 <?php
 
-
-require_once 'CiviTest/CiviUnitTestCase.php';
-
 /**
  * Class Api_TaskTest
  */
 class api_v3_TaskTest extends CiviUnitTestCase {
-  private $_taskTypeId = null;
 
-  function setUp() {
+  /**
+   * @var int
+   */
+  private $_taskTypeId;
+
+  /**
+   * Runs the upgrader and creates task type required for test
+   */
+  public function setUp() {
     parent::setUp();
     $upgrader = CRM_Tasksassignments_Upgrader::instance();
     $upgrader->install();
 
-    // Pick one of Task types.
-    $taskTypes = civicrm_api3('Task', 'getoptions', array(
-      'field' => 'activity_type_id',
-    ));
-    $this->_taskTypeId = array_shift($taskTypes['values']);
-  }
+    $result = civicrm_api3('OptionValue', 'create', [
```

Comment:
Unless you have a good reason not to, you should use the `tearDown` method to deleted anything you create on a `setUp` method.